### ISSUE TITLE
New update alert on my ax5400 flashed to ax82u.  Do it from there or file?

### ISSUE THREAD

**Issue by dfgdgdgfd** [gnuton/asuswrt-merlin.ng#262]
Is this normal for non betas?  I don't know if it's ofw or not.

**Comment by gnuton** [comment]
not sure I got the problem.
I guess you have Aimesh with Ax5400 and Ax82.
I think if you have mixed environemnts OFW and CFW the CFW will update to CFW and the OFW will update to OFW but I am not sure about how the notifications would work.

**Comment by dfgdgdgfd** [comment]
I don't have aimesh devices but I guess the router is capable of using them.

**Comment by gnuton** [comment]
I apologie I though you were talking about the TUF-AX5400.
My mistake. You got an update alert because tehre is a new DSL-AX82U version available.

**Comment by dfgdgdgfd** [comment]
Yes, I'm just not sure whether I can directly update from within the router or whether I have to download the update file from here and point the router to it.

**Comment by gnuton** [comment]
If you have flashed my firmware the update notification is for a new version of my firmware. 
Merlin fimware must be downloaded and manually updated. So you cannot go wrong.
